src/CartesianProduct.jl

```diff
@@ -353,6 +370,53 @@ function ∈(x::AbstractVector{N}, cpa::CartesianProductArray{N, <:LazySet{N}}
     return true
 end
 
+"""
+    constraints_list(cpa::CartesianProductArray{N})::Vector{LinearConstraint{N}} where N<:Real
+
+Return the list of constraints of a (polytopic) Cartesian product.
+
+### Input
+
+- `cpa` -- Cartesian product
+
+### Output
+
+A list of constraints.
+
+"""
+function constraints_list(cpa::CartesianProductArray{N})::Vector{LinearConstraint{N}} where N<:Real
```

Comment:
I think we should better use the interface `constraints_list(cpa::CartesianProductArray{N, <:AbstractPolytope}` given the recent introduction of `constraints_list` list method [for the whole interface](https://github.com/JuliaReach/LazySets.jl/pull/734).

There is a drawback  in this approach (but i don't have a clear picture how to make this play nicely with interfaces), in that the interface restriction i propose will make `constraints_list` of, say, the lazy linear map of a polytope not work. But first we should add `constraints_list(::LazySets.LinearMap{Float64,LazySets.HPolytope{Float64},Float64,Array{Float64,2}})` anyways.